### PR TITLE
Add support for MonetDB using the JDBC adapter

### PR DESCRIPTION
**Pull request by vogti**
This PR adds support for the column store MonetDB.

### CODE REVIEW

File: core/src/main/java/ch/unibas/dmi/dbis/polyphenydb/sql/dialect/MonetdbSqlDialect.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+@Slf4j
+public class MonetdbSqlDialect extends SqlDialect {
+
+    public static final SqlDialect DEFAULT = new MonetdbSqlDialect(
+            EMPTY_CONTEXT.withDatabaseProduct( DatabaseProduct.MONETDB ).withIdentifierQuoteString( "\"" )
+    );
+
+
+    /**
+     * Creates an HsqldbSqlDialect.
```
Comment (schoenja) [inline core/src/main/java/ch/unibas/dmi/dbis/polyphenydb/sql/dialect/MonetdbSqlDialect.java]:
```suggestion
     * Creates an MonetdbSqlDialect.
```

File: dbms/build.gradle [hunk 2]
```diff
@@ -27,6 +27,9 @@ repositories {
         // DBIS Nexus
         url "https://dbis-nexus.dmi.unibas.ch/repository/maven2/"
     }
+    maven {
```
Comment (schoenja) [inline dbms/build.gradle]: Is this required here? There are no other changes to the build file which would use this repository afaict.

File: jdbc-adapter/src/main/java/ch/unibas/dmi/dbis/polyphenydb/adapter/jdbc/stores/MonetdbStore.java [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+            connectionFactory.close();
+        } catch ( SQLException e ) {
+            log.warn( "Exception while shutting down {}", getUniqueName(), e );
+        }
+    }
+
+
+    @Override
+    protected void reloadSettings( List<String> updatedSettings ) {
+        // TODO: Implement disconnect and reconnect to PostgreSQL instance.
```
Comment (schoenja) [inline jdbc-adapter/src/main/java/ch/unibas/dmi/dbis/polyphenydb/adapter/jdbc/stores/MonetdbStore.java]:
```suggestion
        // TODO: Implement disconnect and reconnect to MonetdbSQL instance.
```

File: core/src/main/java/ch/unibas/dmi/dbis/polyphenydb/sql/dialect/MonetdbSqlDialect.java [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+import ch.unibas.dmi.dbis.polyphenydb.sql.SqlCall;
+import ch.unibas.dmi.dbis.polyphenydb.sql.SqlDialect;
+import ch.unibas.dmi.dbis.polyphenydb.sql.SqlNode;
+import ch.unibas.dmi.dbis.polyphenydb.sql.SqlWriter;
+import lombok.extern.slf4j.Slf4j;
+
+
+/**
+ * A <code>SqlDialect</code> implementation for the Hsqldb database.
```
Comment (schoenja) [inline core/src/main/java/ch/unibas/dmi/dbis/polyphenydb/sql/dialect/MonetdbSqlDialect.java]:
```suggestion
 * A <code>SqlDialect</code> implementation for the Monetdb database.
```

File: dbms/build.gradle (hunk 2 above)
Comment (schoenja) [reply dbms/build.gradle]: Moved to #166

**Comment by vogti** [comment]
Thx @schoenja for reviewing this PR.

**Pull request merged**